src/backend/utils/extract-urls.js

```diff
@@ -1,10 +1,14 @@
-// The goal is to create a function that given a html blog
-// it will parse the URLs from that feed and return an array of urls
-// Later on , you can then organize the types of urls (github etc)...
+/*
+  this function returns an array of urls from the html or
+  returns null if there were no urls in the blog feed data
+*/
 const getUrls = require('get-urls');
-// this function returns an array of urls from the html
+
 exports.extract = function(htmlData) {
   const urls = getUrls(htmlData);
   const arrayOfUrls = Array.from(urls);
+  if (arrayOfUrls.length === 0) {
```

Comment:
I would suggest returning an empty array is better than `null`, as it will mean that callers don't have to do an extra `if` check before trying to iterate.  If there are no elements in the array, it won't cause any problems, since the loops won't run.